src/models/city.js

```diff
@@ -0,0 +1,30 @@
+module.exports = (sequelize, DataTypes) => {
+  const City = sequelize.define("Cities", {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    gallary: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    location: {

```

Comment:
change this location field to `capital`, now, so instead of this model called city, call it **state**, now its a state migration.